ide/api/ycm.py

```diff
@@ -60,7 +61,7 @@ def _spin_up_server(request):
             if result.ok:
```

Comment:
I've tested this with SSL now and my own self-signed certs, but I had to set verify=False to stop my virtual machine's CloudPebble complaining. I assume production CloudPebble should be fine since the certificates are all legit?